### ISSUE TITLE
`maji new` fails on OS X 10.11

### ISSUE THREAD

**Issue by bazzel** [kabisa/maji#76]
```bash
➜  my-dir maji new org.example.my-awesome-app my-app
Setting up my-app
phantomjs: This formula either does not compile or function as expected on OS X
versions newer than Yosemite due to an upstream incompatibility.
Error: An unsatisfied requirement failed this build.
```

Expect this to be related with https://github.com/Homebrew/homebrew/issues/42249.

Maji checks if phantomjs is installed using homebrew. It then tries to install phantomjs using homebrew which shows the error.

To work around this you can comment out the check:
```
# /usr/local/lib/node_modules/maji/project_template/bin/setup
brew_dep_installed 'ant' || brew install ant
#brew_dep_installed 'phantomjs' || brew install phantomjs
```

**Comment by eirvandelden** [comment]
PhantomJS 2.0.0 indeed doesn't install on El Capitan. 2.0.1 fixes this, there is a pre-built version available, but no solution using Homebrew, until they release 2.0.1 :(

**Comment by pascalw** [comment]
Something that could fix this and would be better in general is to change the setup script such that it checks for `phantomjs` in your `$PATH` and don't try to install if it's there. That way you can get it from anywhere, (heck self compile if you like) and it won't try to reinstall.

**Comment by rrooding** [comment]
Fixed in #86 with temporary solution
